website/home/app/legal/privacy-policy/privacy-policy.md

```diff
@@ -0,0 +1,261 @@
+# **Privacy Policy**
+
+### **Effective Date: February 19, 2025**
+
+At GenSX, Inc. (herein, “GenSX”, “we” or “Company”), we take your privacy seriously. Please read this Privacy Policy to learn how we treat your personal data. By using or accessing our [website](http://gensx.com) (herein, the “Website”) or services, you acknowledge that you accept the practices and policies outlined below, and you hereby consent that we will collect, use and share your information as described in this Privacy Policy.
+
+Remember that your use of our SaaS-based services (as defined within our Terms of Service) is at all times subject to our [Terms of Service](https://www.moment.dev/terms-of-service), which incorporates this Privacy Policy as applicable. Any terms we use in this Policy without defining them have the definitions included within our [Terms of Service](https://www.moment.dev/terms-of-service).
```

Comment:
These should point to gensx.com links for terms of service.

Reply:
Ah good catch, resolved.